### PR TITLE
OBSIDIAN - Lockscreen Background & Round - Updates

### PR DESCRIPTION
**Pull request by Ian-Boraks**
Made the lockscreen background adhere to the obsidian theme.

```css
.lockscreen_Container_3rFFU {
  background: var(--obsidian-main-color) !important;
}
```

### CODE REVIEW

**Comment by Ian-Boraks** [comment]
Filter Rounding:
Normal View
Advanced View

---

Controller Setting Rounding
Browse Community
Layouts

File: Round/shared.css [hunk 1]
```diff
@@ -43,6 +43,15 @@ button.gamepaddialog_Button_1kn70.DialogButton:enabled:active:hover.Primary {
...
 }
+.steamdeckcompatfilter_CompatFilterDialogRow_O7cW3:nth-child(1) {
+  border-radius: 20px 20px 0px 0px;
+}
+.steamdeckcompatfilter_CompatFilterDialogRow_O7cW3:nth-child(4) {
+    border-radius: 0px 0px 20px 20px;
+}
+.steamdeckcompatfilter_CompatFilterDialogRow_O7cW3:last-child {
+    border-radius: 20px;
+}
```
Comment (Ian-Boraks) [inline Round/shared.css:54]: These are for the filter changes

File: Round/shared.css [hunk 2]
```diff
@@ -236,10 +245,17 @@ button.gamepaddialog_Button_1kn70.DialogButton:enabled:active:hover.Primary {
   border-radius: 20px !important;
 }
 
-.controllerconfigurator_interstitial_ControlLabel_tGJeF {
+.controllerconfigurator_interstitial_ControlLabel_tGJeF,
+.controllerconfiguratorchooseconfiguration_ConfigurationButton_2luCN, /* This is selecting a controller layout */
+.controllerconfigurator_CommunityConfigsCallout_1xfC8  {
   border-radius: 20px;
 }
```
Comment (Ian-Boraks) [inline Round/shared.css:252]: Controller Layouts

**Comment by Ian-Boraks** [comment]
Rounded Friends in Search
Rounded Friends in Search

File: Round/shared.css [hunk 3]
```diff
@@ -140,6 +149,10 @@ button.gamepaddialog_Button_1kn70.DialogButton:enabled:active:hover.Primary {
   border-radius: 20px;
   overflow: hidden;
 }
+.gamepadsearch_ResultTemplateImage_3K3Bx {
+  border-radius: 20px;
+  overflow: hidden;
+}
```
Comment (Ian-Boraks) [inline Round/shared.css:155]: Rounded friends in search

**Review by EMERALD0874 — CHANGES_REQUESTED** [review]
Lock screen fix for Obsidian has already been made. Please remove this from your PR as it would create redundant styling.

**Comment by Ian-Boraks** [comment]
Reverted Commit for Lock-screen in Obsidian

**Review by EMERALD0874 — CHANGES_REQUESTED** [review]
For consistency with the library filters, could you make each individual section of the game options (play through developer, properties, cancel) rounded? I know I said I wasn't a huge fan when I tested it before but I think I've been swayed into liking it.

Also, do you mind increasing the theme.json one minor version to 1.3?

**Review by EMERALD0874 — APPROVED** [review]
LGTM

**Pull request merged**